source/main.cpp

```diff
@@ -95,7 +95,8 @@ int main(int argc, char *argv[])
 	// Ensure that we log errors to the errors.txt file.
 	Logger::SetLogErrorCallback([&hasErrors](const string &errorMessage) {
 		static const string PARSING_PREFIX = "Parsing: ";
-		if(errorMessage.substr(PARSING_PREFIX.length()) != PARSING_PREFIX)
+		if(errorMessage.length() <= PARSING_PREFIX.length() || errorMessage.substr(PARSING_PREFIX.length())
+				!= PARSING_PREFIX)
```

Comment:
```suggestion
		if(errorMessage.substr(std::min(PARSING_PREFIX.length(), errorMessage.length()))
				!= PARSING_PREFIX)
```
what about this?

Reply:
I guess it is the same

Reply:
Honestly, I don't know which one is better... if we go with your solution, do we still need the line break?

Reply:
but we less than 120

Reply:
It's going in, then.

Reply:
```suggestion
		if(errorMessage.substr(min(PARSING_PREFIX.length(), errorMessage.length())) != PARSING_PREFIX)
```